app/controllers/idv/in_person/validate_transliterable_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Idv
+  module InPerson
+    class ValidateTransliterableController < ApplicationController
+      include RenderConditionConcern
+      include UspsInPersonProofing
+      include EffectiveUser
+      include UspsInPersonProofing
```

Comment:
dublicate

Reply:
also, why include the whole namespace? which classes do we use from it, and could we just use their long names instead?

Reply:
Thanks, I'll remove the duplicate. Including the whole namespace isn't strictly necessary, so I can remove it.